src/main/groovy/com/github/benmanes/gradle/versions/updates/DependencyUpdatesTask.groovy

```diff
@@ -88,6 +88,20 @@ class DependencyUpdatesTask extends DefaultTask {
     group = 'Help'
 
     outputs.upToDateWhen { false }
+
+    if(supportsIncompatibleWithConfigurationCache()) {
+      callIncompatibleWithConfigurationCache()
+    }
+  }
+
+  private boolean supportsIncompatibleWithConfigurationCache() {
+    return metaClass.respondsTo(this, "notCompatibleWithConfigurationCache", String)
+  }
+
+  private void callIncompatibleWithConfigurationCache() {
+    String methodName = "notCompatibleWithConfigurationCache"
+    Object[] methodArgs = ["The gradle-versions-plugin isn't compatible with the configuration cache"]
+    metaClass.invokeMethod(this, methodName, methodArgs)
```

Comment:
I think you can simply invoke the method since it is dynamically typed? Our other usages only guard before making the method call directly.

Reply:
I'm not the most well versed on Groovy because that's what I thought as well, but the IDE was showing it as red. I didn't actually try running it though.

Reply:
haha, me neither. I just use VSCode and pray =)

Reply:
I just double checked and it won't run. I'm assuming the compiler can't see it because the version of the Gradle API needs to be bumped, but not sure if that is desired at the moment.

Reply:
Sounds good, thanks!